mir-ci/mir_ci/robot_libraries/WaylandHid.py

```diff
@@ -29,12 +30,15 @@ async def move_pointer_to_absolute(self, x: float, y: float) -> None:
         """Move the virtual pointer to an absolute position within the output."""
         await self.connect()
         self.move_to_absolute(x, y)
+        self.pointer_position = {"x": x, "y": y}
 
     @keyword
     async def move_pointer_to_proportional(self, x: float, y: float) -> None:
         """Move the virtual pointer to a position proportional to the size of the output."""
         await self.connect()
         self.move_to_proportional(x, y)
+        absolute_position = await self.get_absolute_from_proportional(x, y)
+        self.pointer_position = absolute_position
```

Comment:
Let's just return a tuple from `move_to_proportional`?

```suggestion
        self.pointer_position["x"], self.pointer_position["y"] = self.move_to_proportional(x, y)
        return self.pointer_position
```